ui/spells/spells.scss

```diff
@@ -100,42 +105,30 @@
 .spell-note {
   flex-grow: 1;
   margin-bottom: 1px;
-  margin-left: 50px;
 }
-
-.spell-note.spell-note.spell-note.spell-note {
-  text-align: left;
-  margin-top: -1px;
-}
-
 .arcane .spell-note {
-  margin-left: 90px;
+  margin-left: 40px;
 }
 
 /* Avoid space between items in the row */
-.spells .table-header {
-  display: flex;
-  flex-direction: row;
-  align-items: flex-end;
-}
-
-.spells div[data-groupname]>div {
-  display: flex;
-  flex-direction: column;
-  align-items: stretch;
-  /* and make rows overlap */
-  margin-top: -1px;
+.spells {
```

Comment:
No change, just nesting.

Reply:
Review without whitespace to see:
https://github.com/epic-power-rpg/roll20/pull/23/files?diff=unified&w=1